src/Octoshift/Services/EnvironmentVariableProvider.cs

```diff
@@ -56,16 +57,27 @@ public EnvironmentVariableProvider(OctoLogger logger)
     public virtual string SmbPassword(bool throwIfNotFound = true) =>
         GetSecret(SMB_PASSWORD, throwIfNotFound);
 
-    private string GetSecret(string secretName, bool throwIfNotFound)
+    public virtual string SkipStatusCheck(bool throwIfNotFound = false) =>
+        GetValue(GEI_SKIP_STATUS_CHECK, throwIfNotFound);
+
+    private string GetValue(string name, bool throwIfNotFound)
     {
-        var secret = Environment.GetEnvironmentVariable(secretName);
+        var value = Environment.GetEnvironmentVariable(name);
 
-        if (string.IsNullOrEmpty(secret))
+#pragma warning disable IDE0046 // Convert to conditional expression
+        if (string.IsNullOrEmpty(value))
```

Comment:
nit: `value.IsNullOrWhiteSpace()`

Reply:
Addressed this in 13ba4f10a2a2318beeb4acc7519ffdf9e5ff5441

Since this changes the env variable semantic slightly also added a unit test